contracts/example_components/unpermissioned/README.adoc

```diff
@@ -0,0 +1,13 @@
+= Unpermissioned example components
```

Comment:
rename contracts/example_components to contracts/examples

Reply:
done